### ISSUE TITLE
Develop framing around turn-key options

### ISSUE THREAD

**Issue by kestlund** [projecthydra-labs/power-steering#3]

**Comment by kestlund** [comment]
HyBox, Sufia, Avalon, etc.  What is needed?

**Comment by mark-dce** [comment]
We need a better resource for new adopters asking "which hydra head should I use?"  

Possible resources could be:
- a feature comparison matrix such as the one here: https://wiki.duraspace.org/display/hydra/Feature+Inventory+Grid
- a deployment guide - or at least a consolidated document pointing to hydra-heads that have canned deployment scripts or VMs
- A set of clarifying questions to help adopters think about their needs - e.g.:
  - Is your use focused on self deposit or administrative management
  - What type of content do you need to manage (images, documents, media, etc.) and how much of each?
  - How big is your collection, how rabidly is it growing?
  - etc.
- how to find technical training
- how to find manager oriented training

Much of this information exists in bits and pieces, but my be difficult for new adopters to easily find.

**Comment by jcoyne** [comment]
I think the feature inventory grid is overwhelming. It wouldn't be the first thing I'd want to point an adopter at.

PS. @mark-dce I _think_ you typo'ed  "rabidly", but it kinda works.

**Comment by mark-dce** [comment]
Agreed - the feature matrix is definitely not new-adopter friendly at this point.

**Comment by kestlund** [comment]
@mark-dce Is this still a to-do?
